source/steamshim/src/os.cpp

```diff
@@ -60,17 +71,25 @@ bool setEnvVar(const char *key, const char *val)
 
 bool launchChild(ProcessType *pid, const char* name)
 {
-    char *str = _strdup( GetCommandLineA() );
-
     STARTUPINFOA si = { sizeof( si ) };
     memset( pid, 0, sizeof( *pid ) );
 
     char exename[32] = ".\\";
```

Comment:
Is this `".\\"` needed for `GetFileAttributesA` or `CreateProcessA` ?
I very much doubt it.